iam_user.tf

```diff
@@ -0,0 +1,33 @@
+data "aws_iam_policy_document" "user" {
+  count = "${var.user_enabled == "true" ? 1 : 0}"
+
+  statement {
+    actions = [
+      "logs:DescribeDestinations",
+      "logs:DescribeExportTasks",
+      "logs:DescribeLogGroups",
+      "logs:DescribeLogStreams",
+      "logs:DescribeMetricFilters",
+      "logs:DescribeSubscriptionFilters",
+      "logs:FilterLogEvents",
+      "logs:GetLogEvents",
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+      "logs:DescribeLogStreams",
+      "logs:CreateLogStream",
+      "logs:DeleteLogStream",
+    ]
+
+    resources = ["*"]
+  }
+}
+
+module "user" {
```

Comment:
Do we need this module?
It is not in use.

Reply:
>It is not in use.
 
Why do you decided that?

Reply:
I assume, purpose of this user is to be able to connect to the corresponding streams.
But this user would has access to all streams. https://github.com/cloudposse/terraform-aws-cloudwatch-logs/pull/1/files#diff-a83a8bb953f15707ac8beb549b114ab5R22